server/modules/freckle/projects.js

```diff
@@ -60,20 +60,47 @@ export const createProject = async (projectName) => {
   }
 };
 
+/**
+ * @desc Get a user id with their email address on freckle
+ *
+ * @param {string} email - user email address be assigned to a project
+ *
+ * @returns {promise} - Axios response for request to get userId from email
+ */
+const getUserIdByEmail = async (email) => {
+  const url = `${freckleUrl}/users?freckle_token=${freckleToken}`;
```

Comment:
You can actually pass the `email` as a query string in the `url` like this:
```
const url = `${freckleUrl}/users?freckle_token=${freckleToken}?email=${email}`;
```
Hitting this endpoint would return that particular user with the email specified, consequently.
You can check the `parameters` section in the documentation of the `users` freckle endpoint for more info: [link](http://developer.letsfreckle.com/v2/users/#list-users)